coalib/bears/Bear.py

```diff
@@ -232,7 +254,8 @@ def maintainers_emails(cls):
     def __init__(self,
                  section: Section,
                  message_queue,
-                 timeout=0):
+                 timeout=0,
+                 debug_flag=False):
```

Comment:
Just call it `debug`, the `_flag` part is redundant, any boolean argument can be called a "flag" ;)